tests/custom_pk/tests.py

```diff
@@ -230,3 +231,13 @@ def test_required_pk(self):
         with self.assertRaises(IntegrityError):
             with transaction.atomic():
                 Employee.objects.create(first_name="Tom", last_name="Smith")
+
+    def test_auto_field_subclass_create(self):
+        instance = AutoModel.objects.create()
+        self.assertIsInstance(instance.id, MyIntWrapper)
+
+    @skipUnlessDBFeature('can_return_rows_from_bulk_insert')
+    def test_auto_field_subclass_bulk_create(self):
+        instance = AutoModel()
+        AutoModel.objects.bulk_create([instance])
+        self.assertIsInstance(instance.id, MyIntWrapper)
```

Comment:
This is already covered by `tests.queries.test_db_returning.ReturningValuesTests.test_bulk_insert`, that's why `ReturningValuesTests` tests crash on SQLite 3.35+ (see #14227).

Reply:
However, I think it's worth to keep it for backends without built-in converters.